src/main/python/package_bundles.py

```diff
@@ -215,7 +214,7 @@ def prepare_mgmt_bundle(mgmt):
 def prepare_policies(version):
     print("==================================")
     print("Prepare Policies")
-    policies_dist_path = "%s/dist/graviteeio-policies-%s" % (tmp_path, version)
+    policies_dist_path = "%s/dist/gravitee-policies-%s" % (tmp_path, version)
```

Comment:
it's a little bit weird I know, but later on the script I rename directories (because we have named distributions with gravitee-...